IdentityCore/src/controllers/MSIDRequestControllerFactory.m

```diff
@@ -43,6 +48,30 @@ @implementation MSIDRequestControllerFactory
                                                          skipLocalRt:(MSIDSilentControllerLocalRtUsageType)skipLocalRt
                                                 tokenRequestProvider:(id<MSIDTokenRequestProviding>)tokenRequestProvider
                                                                error:(NSError *__autoreleasing*)error
+{
+    if (parameters.msidXpcMode != MSIDXpcModeDisable)
+    {
+        return [self v2SilentControllerForParameters:parameters
```

Comment:
IMO, I would avoid using v1, v2 etc in the naming of things. As time goes by what the versions mean relies of anecdotal knowledge and gets lost. Also, this implies that there will be new versions in the future like v3, v4 etc.

I would suggest renaming v2SilentControllerForParameters: as xpcEnabledSilentControllerForParameters or add a new argument like isXpcEnabled to the original method.

Reply:
Updated the method name. The ending goal is to remove the `v1` and to use `v2`. For now I changed the name to reflect the purpose